ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/AmountRowMapper.java

```diff
@@ -2,12 +2,12 @@
 
 import org.springframework.stereotype.Component;
 import uk.gov.hmcts.cmc.ccd.domain.CCDAmountRow;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCollectionElement;
 import uk.gov.hmcts.cmc.domain.models.AmountRow;
 
 @Component
-public class AmountRowMapper implements Mapper<CCDAmountRow, AmountRow> {
+public class AmountRowMapper {
 
-    @Override
     public CCDAmountRow to(AmountRow amountRow) {
```

Comment:
Shouldn't we return `CCDCollectionElement<CCDAmountRow>` for `to(..)` to be consistent here?

Reply:
it was needed , done now